### DISCUSSION TITLE
Docker Compose Pull Access Denied?

### DISCUSSION THREAD

**Discussion by peterhoang** [danny-avila/LibreChat#2521] [Troubleshooting]
Not sure if I'm the only one, but I've been getting access denied on  `docker pull ghcr.io/danny-avila/librechat-dev:latest`. When I searched docker hub, I found https://hub.docker.com/r/shaoyun/librechat-dev-api which I can pull successfully.

**Comment by danny-avila** [comment]
Seems like a GHCR issue: https://stackoverflow.com/questions/74795993/unable-to-pull-public-image-from-github-container-registry-ghcr

**Comment by peterhoang** [comment]
Turns out my PAT expired. I regenerated the token and relogged in. See https://stackoverflow.com/questions/65072379/docker-login-access-denied-you-must-use-a-personal-access-token
